src/main/webapp/js/instructorFeedbackEdit.js

```diff
@@ -651,21 +651,29 @@ function disableRow(elem, row) {
     $tdElements.parent().hide();
 }
 
-function feedbackRecipientUpdateVisibilityOptions(elem) {
-    elem = $(elem);
-    if (elem.val() === 'OWN_TEAM' || elem.val() === 'TEAMS' ||
-            elem.val() === 'INSTRUCTORS' || elem.val() === 'OWN_TEAM_MEMBERS') {
-        enableRow(elem, 1);
-        disableRow(elem, 3);
+function feedbackRecipientUpdateVisibilityOptions(recipientType) {
+    $recipientType = $(recipientType);
+    if (isRecipientsTeamMembersValidVisibilityOptionForRecipientType($recipientType.val())) {
+        // show the row Recipient(s) and hide the row Recipient's Team Members
+        enableRow($recipientType, 1);
```

Comment:
Your comment says "show", but the function is called "enable"... Maybe we need to make a new issue to rename the function?